src/parser/DefaultParser.js

```diff
@@ -24,6 +26,27 @@ class DefaultParser {
     this.pluginData.parseErrors = [];
   }
 
+  /**
+   * Set configuration into Components.
+   * @param {FileInput} configFile - File input to retrieve configuration.
```

Comment:
put the same def for every `configFile` param

Reply:
done

Reply:
nope